.travis.yml

```diff
@@ -33,6 +33,8 @@ install:
       pip install -r requirements2.txt;
     else
       pip install -r requirements3.txt;
+      wget http://launchpadlibrarian.net/151932048/libc6_2.17-0ubuntu5.1_i386.deb;
```

Comment:
@godber Didn't work. PyQt5 is updated to v9 and then I got this problem: https://travis-ci.org/planetarypy/pdsspect/jobs/250553104